Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,10 +1,170 @@
+
+  * Disabled GPU Skinning to optimize GPU usage and FPS.
+  * Lowered quality of ambient occlusion from high to medium.
+  * Switched SMAA High to FXAA (fast mode) reducing GPU cost.
+  * Modified GPU Instancing on imps, heroes, and the boss to significantly reduce the number of draw calls.
+  * Turned off Cast Shadows on Imp and Imp Boss.
+  * Disabled mesh colliders of lava, which is more decorative than interactive.
+  * Refactored the S_SimpleDissolve shader which consumed most import time.
+
+* Updated the Photon Setup Guide, indicating you need only app ID when playing with friends. For users connecting across regions, you may need to hard code a region in your app settings by using the room code and region instead of just the room code sharing in game. 
+* Removed a duplicated `GameObject` from the MainMenu scene. 
+* Reviewed and revised code to better following quality standards.
+* Updated the Mage character base attack to better support the new enqueuing ability and handle game behaviors. Updates include:
+
+  * Actions being non-blocking now allow other actions while a mage-bolt is in flight
+  * Actions send in `ActionSequence` groups to better handle actions when players spam click enemies
+  * Timing issues with animations, actions, and character location
+  * Bolt animation for legitimate hits
+  * Updated attacks not to knock rogues out of stealth 
+
+* Updated character attacks to not cause friendly-fire damage between players. 
+* Updated and resolved issues with 3D models including polygon count of coins and chest, artifacts where level graphics are stitched together or overlapping characters, and asset map consistency used by objects (color + normal map + emission).
+* Merged `ConnectStatus` and `DisconnectReason` into a single `ConnectStatus`. 
+* Updated `ServerGameNetPortal` to properly handle the following:
+
+  * Per-connection state on client disconnect
+  * Additional errors including a full server and player ID (GUID) already playing
+  * Player connection if they experience a game-level connection failure
+
+* Updated code to allow hosts to specify a port to listen to, removing the hard-coded port. 
+* Refactored Action Bar code including the following:
+
+  * Removed the `ButtonID` from `UIHudButton`.
+  * Removed hard-coded values from `HeroActionBar`.
+  * Removed switch statements.
+  * Completed minor code cleanup.
+  * Added verification to only show skill and ability buttons for available character abilities. Empty buttons no longer load for characters.
+
+* Added a call to warm up shaders when the project starts to ensure animations issues do not occur.
+* Removed collision from objects that have a Broken (dead) state.
+* Implemented a better cooldown solution and calculations for tracking and managing character, imp, and boss actions. 
+* Updated event registration and unregistration code to be symmetrical across the project.
+
+### Fixes
```

Comment:
It feels like a lot of the above are fixes too.

Reply:
Will look them over for best location as update vs fix, thanks!!